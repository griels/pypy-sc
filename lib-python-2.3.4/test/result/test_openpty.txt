command: /usr/bin/python /home/arigo/pypysrc/pypy/tool/alarm.py 900 /home/arigo/pypysrc/pypy/interpreter/py.py --oldstyle /home/arigo/pypysrc/lib-python-2.3.4/test/test_openpty.py
run by: arigo@tismerysoft.de
sys.platform: linux2
sys.version_info: (2, 3, 5, 'final', 0)
startdate: Wed Apr 27 22:57:19 2005
pypy-revision: 11518
OUTPUT TEST
see output in: /home/arigo/pypysrc/lib-python-2.3.4/test/result/test_openpty.out
============================================================
faking <type 'unicode'>
faking <type 'module'>
faking <type 'file'>
faking <type 'posix.stat_result'>
faking <type 'posix.statvfs_result'>
========================== closed ==========================
execution time: 5.74162387848 seconds
exit status: 0
