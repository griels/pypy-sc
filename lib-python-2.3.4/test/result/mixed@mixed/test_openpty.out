test_openpty
Calling os.openpty()
(master, slave) = (3, 4)
Ping!
